ui/component/viewers/videoViewer/view.jsx

```diff
@@ -204,6 +204,11 @@ function VideoViewer(props: Props) {
     if (player) {
       player.on('tracking:buffered', (e, d) => doTrackingBuffered(e, d));
       player.on('tracking:firstplay', (e, d) => doTrackingFirstPlay(e, d));
+
+      // fixes #3498 (https://github.com/lbryio/lbry-desktop/issues/3498)
+      // summary: on firefox the focus would stick to the fullscreen button which caused buggy behavior with spacebar
+      // @ts-ignore (adding ts-ignore due to the ".blur()" throwing an issue with typescript)
```

Comment:
I think you want `$FlowFixMe`, instead of `@ts-ignore`?

Reply:
Changed to `FlowFixMe`